src/code/graph.c

```diff
@@ -222,8 +222,8 @@ void Graph_TaskSet00(GraphicsContext* gfxCtx) {
 
     scTask->next = NULL;
     scTask->flags = OS_SC_NEEDS_RSP | OS_SC_NEEDS_RDP | OS_SC_SWAPBUFFER | OS_SC_LAST_TASK;
-    if (SREG(33) & 1) {
-        SREG(33) &= ~1;
+    if (R_GRAPH_TASKSET00_FLAGS & 1) {
+        R_GRAPH_TASKSET00_FLAGS &= ~1;
         scTask->flags &= ~OS_SC_SWAPBUFFER;
         gfxCtx->fbIdx--;
     }
```

Comment:
was hoping this reg would make its purpose clear that it is stopping the frame buffer indexing from incrementing and freezing the screen. "task set 00" means nothing to me personally

I do understand that it is a set of flags because of how its set and checked, but it pretty unfortunate lol

Reply:
I would be fine naming the reg after its only flag, but I just don't know what to name it

To me the current name is temporary, it's just for the sake of not being a random REG()

Reply:
Can understand not wanting to do it in this PR, but the `1` could probably use a define which might help with this situation.

Reply:
I would like this PR to get in so yeah I'd rather not add stuff to it now, months later
I'll add it to my todolist though

Btw this PR is a 80 lines diff, my nearing-100% documentation of pause menu is a 6000 lines diff right now...